src/Nox.Generator/Application/Commands/PartialUpdateOwnedCommand.template.cs

```diff
@@ -70,7 +70,7 @@ internal abstract class PartialUpdate{{entity.Name}}For{{parent.Name}}CommandHan
 			return null;
 		}
 
-		EntityFactory.PartialUpdateEntity(entity, request.UpdatedProperties);
+		EntityFactory.PartialUpdateEntity(entity, request.UpdatedProperties, DefaultCultureCode);
```

Comment:
Owned Entities need also to support localized text fields
We are creating a ticket for handling this